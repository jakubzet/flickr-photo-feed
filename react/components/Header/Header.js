import React from "react";
import * as S from "./styled";

export const Header = () => (
  <S.Header>
    <S.Wrapper>
      <S.Logo href="/">
        <h1>Flickr Feeds Fetcher</h1>
      </S.Logo>
      <S.SearchWrapper>
        <S.Search type="text" placeholder="Filter feed..." autocomplete="off" />
      </S.SearchWrapper>
    </S.Wrapper>
  </S.Header>
);
