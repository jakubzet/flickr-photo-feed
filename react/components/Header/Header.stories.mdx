import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
import Header from "./Header";

<Meta
  title="Components|Header"
  component={Header}
  parameters={{ jest: ["Header"] }}
/>

# Header

Header description placeholder.

<Preview>
  <Story name="default">
    <Header />
  </Story>
</Preview>

<Props of={Header} />