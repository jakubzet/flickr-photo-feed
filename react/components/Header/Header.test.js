import React from "react";
import { render } from "../../utils/reactTestingLibrary";
import { Header } from "./Header";

describe("Header", () => {
  it("Renders", () => {
    const { container, asFragment } = render(<Header />);

    expect(container).toBeDefined();
    expect(asFragment()).toMatchSnapshot();
  });
});
