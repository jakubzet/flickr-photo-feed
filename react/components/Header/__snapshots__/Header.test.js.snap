// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Renders 1`] = `
<DocumentFragment>
  <header
    class="sc-EHOje iOeApy"
  >
    <div
      class="sc-bdVaJa dgpabP"
    >
      <a
        class="sc-bwzfXH fwsfYo"
        href="/"
      >
        <h1>
          Flickr Feeds Fetcher
        </h1>
      </a>
      <div
        class="sc-htpNat bOYgPy"
      >
        <input
          class="sc-bxivhb fFdMlD"
          placeholder="Filter feed..."
          type="text"
          value=""
        />
      </div>
    </div>
  </header>
</DocumentFragment>
`;
